lib/grid_creator.rb

```diff
@@ -0,0 +1,23 @@
+module GridCreator
+
+  BOX_OF_INDEX = [
+      0,0,0,1,1,1,2,2,2,0,0,0,1,1,1,2,2,2,0,0,0,1,1,1,2,2,2,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.<br>Space missing after comma.